src/i18n/i18n.ts

```diff
@@ -128,13 +128,13 @@ const titleCasePostProcessor: PostProcessorModule = {
 const sentenceCasePostProcessor: PostProcessorModule = {
     type: 'postProcessor',
     name: 'sentenceCase',
-    process: (value: string) => {
+    process: (value: string, _: string, __: TOptions<StringMap>, translator: any) => {
```

Comment:
I'm not a fan of the `_` and `__` variables. Could you rename them to `_key` and `_options` so it's more clear what they're intended to be?